docker-entrypoint.sh

```diff
@@ -2,11 +2,18 @@
 set -e
 
 mkdir -p $TEAMCITY_DATA_PATH/lib/jdbc $TEAMCITY_DATA_PATH/config
-if [ ! -f "$TEAMCITY_DATA_PATH/lib/jdbc/postgresql-9.3-1103.jdbc41.jar" ];
+
+if [ "$DATABASE" == "mysql" ];
+then
+    if [ ! -f "$TEAMCITY_DATA_PATH"/lib/jdbc/mysql-connector-java.jar ];
```

Comment:
Yes, except that my experience with Ubuntu is that they are not maintained that well, for example the `libmysql-java` package is on version [5.1.28](http://packages.ubuntu.com/trusty/all/libmysql-java/filelist) that was released in 12-2013...